fulp_modules/features/antagonists/bloodsuckers/code/powers/vassal/vassal_fold.dm

```diff
@@ -22,12 +22,13 @@
 	. = ..()
 	if(!.)
 		return FALSE
-	var/datum/antagonist/vassal/revenge/revenge_vassal = owner.mind.has_antag_datum(/datum/antagonist/ex_vassal)
-	if(revenge_vassal)
+	var/datum/antagonist/vassal/revenge/revenge_vassal = owner.mind.has_antag_datum(/datum/antagonist/vassal/revenge)
+	if(!revenge_vassal)
```

Comment:
```suggestion
	if(isnull(revenge_vassal))
```

Reply:
arent nulls in DM considered falsey? i dont know what difference this makes aside from explanatory power

Reply:
readability and performance its a nit, not anything that would realistically change anything.

Reply:
It don't matteeeeeeeeerrr